themes/uhsg_theme/templates/views/views-view--search.html.twig

```diff
@@ -64,6 +64,11 @@
       <div id="my-searches">
       </div>
     </div>
+    {% if link_to_instructions_for_students %}
+      <div class="col-right">
+        {{ link_to_instructions_for_students }}
+      </div>
+    {% endif %}
```

Comment:
This addition seem to cause visual glitch with the tooltip positioning.
![image](https://user-images.githubusercontent.com/93661/50345304-41fdc100-0537-11e9-926f-3b174ec0b096.png)

Reply:
HUB-372: Front page tooltip positioning: Position the tooltip closer to the heading. Remove left padding to allow for graceful placement on small screens (can drop below the heading, but remain at the same left padding position with the heading).